test/server.jl

```diff
@@ -36,24 +36,24 @@ end
     socket = connect(port)
     lintbyserver(socket)
     response = ""
-    line = ""
-    while line != "\n"
+    line = "."
+    while !isempty(line)
```

Comment:
My bad, I was late with my review, I thought the lintserver hang was still a problem.